scripts/migrateToPassport.ts

```diff
@@ -0,0 +1,39 @@
+import admin, { ServiceAccount } from "firebase-admin";
+import { FIREBASE_ADMIN_SERVICE_ACCOUNT } from "../config.json";
+import { passportApi } from "../src/services/passport";
+import { User } from "../src/shared/firestoreTypes";
+
+// loop through all users
+
+if (!admin.apps.length) {
+  try {
+    admin.initializeApp({
+      credential: admin.credential.cert(
+        FIREBASE_ADMIN_SERVICE_ACCOUNT as ServiceAccount
+      ),
+    });
+  } catch (error) {
+    console.error("Firebase admin initialization error", error);
+  }
+}
+const db = admin.firestore();
+
+const migrateToPassport = async () => {
+  const usersSnapshot = await db.collection("users").get();
+  const discordIds: string[] = [];
+
+  // For each doc, link on galactic passport and
+  await usersSnapshot.docs.reduce(
+    (p, doc) =>
+      p.then(() => {
```

Comment:
This is sequential right? It will work but it might be a bit slow. It's a 1 time thing though so I'm okay with doing this. Alternatively we could use a map and then `Promise.all` the list of promises?

Reply:
Will the api be able to handle 7000 requests in parallel?